### ISSUE TITLE
Time consumption issue

### ISSUE THREAD

**Issue by chengm0-0** [bairuofei/CGE#1]
Hello, you have provided a very meaningful job. When I was reproducing the code, I found that VRP is a time-consuming process that usually takes more than half a minute on my computer, which hinders the application of work towards incremental graphs. I want to ask, is this a normal phenomenon? thank you very much!

**Comment by bairuofei** [comment]
Hello, thanks for your interest in our work. The code calling VRP_solver is located at [here](https://github.com/bairuofei/CGE/blob/3d5aeb0a8d5fe0c47eeb1b9776a27a05d46ade4d/main.py#L206), where you can find that the time limit for the VRP solver is set to 60 seconds by default. You can try to change this time limit to see if it works for you. 

 As the VRP problem itself is generally NP-Hard, we cannot obtain the optimal solution in polynomial time. However, you can also try different search strategies, like the greedy search as I commented [here](https://github.com/bairuofei/CGE/blob/3d5aeb0a8d5fe0c47eeb1b9776a27a05d46ade4d/VRP_solver.py#L50). Some search strategies run faster, but usually output worse paths for robots. Hope this helps.

**Comment by chengm0-0** [comment]
Thank you for your reply! I have already reduced the VRP time yesterday, and the effect seems to be good at present.
However, there is still a time-consuming problem in transferring this work to multi robot exploration, because the graph composed of target points is updated every moment, and the VRP problem needs to be solved every time. I don't know if my understanding is accurate? Thanks!

**Comment by bairuofei** [comment]
In fact, we assume the prior topo-metric graph of the environment will not change. Therefore, the VRP problem only needs to be solved once in the beginning. However, if the topo-metric graph updates online, you can call the VRP solver again, and then select informative loop closures based on the robots' history pose graphs and their updated VRP paths (a straightforward online extension to our work). 

To summarize, the VRP problem does not need to be solved every time as long as the topo-metric graph does not change.

**Comment by chengm0-0** [comment]
Thank you for your reply. I understand now.
